pcdsdevices/epics/mirror.py

```diff
@@ -1012,26 +1012,38 @@ def inserted(self):
         """
         Whether PointingMirror is inserted
         """
-        return bool(int(self.state.in_state.at_state.value))
+        return self.state.value == 'IN'
 
     @property
     def removed(self):
         """
         Whether PointingMirror is removed
         """
-        return bool(int(self.state.out_state.at_state.value))
+        return self.state.value == 'OUT'
 
     def remove(self, wait=False, timeout=None): 
         """
         Remove the PointingMirror from the beamline
+
+        If the horizontal gantry is not coupled, the request will raise a
+        `RuntimeError`
         """
-        return self.state.move("OUT", wait=wait, timeout=timeout)
+        if self.coupled:
+            return self.state.move("OUT", wait=wait, timeout=timeout)
+        else:
+            raise RuntimeError("Gantry is not coupled, can not move")
```

Comment:
Is this for redundancy, or will the mirror totally allow you to attempt to move one motor super far with the coupling disabled?

Reply:
For redundancy, the mirror will not let you move past a certain gantry difference. Still. Don't want to test this rigorously. Also, the mirror WILL move UP to this gantry difference, so not an ideal situation anyways.

The mirror should never be decoupled though, this was just so for ease of mind.